src/ImageRunModal.jsx

```diff
@@ -483,9 +498,19 @@ export class ImageRunModal extends React.Component {
         if (event === undefined)
             return;
 
+        let command = this.state.command;
+        if (value.Command && !command)
+            command = utils.quote_cmdline(value.Command);
+
+        let entrypoint = "";
+        if (value.Entrypoint)
+            entrypoint = utils.quote_cmdline(value.Entrypoint);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1155-20230104-111228-5fe35f89-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Style -- please make this a `const` and use `? .. :`